test/ApcTokenStorageTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace {  
```

Comment:
I see. Moved to F3\AppNexusClient namespace in 927308ea9a8c29727269a002c2601968084b7ac8